package.json

```diff
@@ -46,31 +46,36 @@
   "devDependencies": {
     "@sinonjs/fake-timers": "github:sinonjs/fake-timers#0bfffc1",
     "@types/node": "^15.3.1",
+    "babel-eslint": "^10.1.0",
     "convert-hrtime": "^5.0.0",
     "cross-zip": "^4.0.0",
     "dedent": "^0.7.0",
     "deepmerge": "^4.2.2",
     "dezalgo": "^1.0.3",
+    "eslint": "^7.32.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-prettier": "^4.0.0",
     "fast-deep-equal": "^3.1.3",
     "into-stream": "^6.0.0",
     "js-yaml": "^4.1.0",
     "license-checker": "^25.0.1",
     "minimist": "^1.2.5",
     "node-fetch": "^2.6.1",
     "ora": "^5.4.0",
+    "prettier": "^2.6.2",
     "pretty-hrtime": "^1.0.3",
     "proxy": "^1.0.2",
     "rimraf": "^3.0.2",
     "semver": "^7.3.5",
     "simple-git": "^3.5.0",
     "simple-statistics": "^7.7.0",
     "split2": "^3.2.2",
-    "standard": "^16.0.3",
     "stoppable": "^1.1.0",
     "tap": "^15.0.9",
     "tsd": "^0.15.1",
     "workq": "^3.0.0",
-    "xmlbuilder2": "^2.4.1"
+    "xmlbuilder2": "^2.4.1",
+    "yarn": "^1.22.18"
```

Comment:
Shouldn't this go under the `engines` property?

Reply:
right now we only use yarn for lint. so I added it to devDependency.

Reply:
We have a `yarn.lock` file though so we can't only be using it for this?

Reply:
will do